src/components/_atoms/Modal/Modal.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react'
+import './Modal.scss'
+
+interface ModalProps {
+  title: string
+  isOpen: boolean
+  onClose: () => void
+  children: ReactNode
+}
+
+export const Modal = ({ title, isOpen, children }: ModalProps) => {
+  return isOpen ? (
+    <div className="modal">
+      <div className="modal_wrapper" />
```

Comment:
why are you using custom modal? What about modal from antd and it is not a atom, modal consist of header, buttos, it can be molecules or organisms